pyproject.toml

```diff
@@ -76,6 +76,9 @@ dependencies = [
     "openai-whisper >= 20231117",
     "django-phonenumber-field == 7.3.0",
     "phonenumbers == 8.13.27",
+    "aiohttp ~= 3.9.0",
+    "beautifulsoup4 ~= 4.12.3",
```

Comment:
Can you also remove the `bs4` dependencies, and make sure the HTML parsing for plaintext indexing still works after removing it and using beautifulsoup4 directly? According to the [pypi project for bs4](https://pypi.org/project/bs4/),

> This is a dummy package designed to prevent namesquatting on PyPI. You should install [beautifulsoup4](https://pypi.python.org/pypi/beautifulsoup4) instead.

note: there are two bs4 dependencies for some reason.

Reply:
Good catch, there were also 2 aiohttp dependencies. Cleaned both up in `pyproject.toml`